data/json/items/gun/38super.json

```diff
@@ -4,7 +4,7 @@
     "copy-from": "pistol_base",
     "looks_like": "glock_17",
     "type": "GUN",
-    "name": { "str_sp": "AF2011A1 .38 Super" },
+    "name": { "str": "AF2011A1 .38 Super pistol" },
```

Comment:
Could drop the caliber designation from the name here as well, it adds unneeded information, the only version of this pistol that exists in the game is in .38 Super.